lab-gurpreet/lib/cowsay.js

```diff
@@ -0,0 +1,7 @@
+var cowsay = require('cowsay');
```

Comment:
This file is unnecessary because you're not using it anywhere, you can get rid of it